internal/services/compute/shared_image_version_resource.go

```diff
@@ -330,6 +395,17 @@ func resourceSharedImageVersionUpdate(d *pluginsdk.ResourceData, meta interface{
 		payload.Properties.PublishingProfile.ExcludeFromLatest = pointer.To(d.Get("exclude_from_latest").(bool))
 	}
 
+	if d.HasChange("uefi_settings") {
```

Comment:
could you add a testcase for the `update` operation?

Reply:
While designing the additional test case I noticed that I get a:
`PropertyChangeNotAllowed: Changing property 'galleryImageversion.properties.securityProfile.uefiSettings' is not allowed.`
I already knew that partially from reading the reference and had a `ForceNew: true` for `uefi_settings`.
I removed the `d.HasChange("uefi_settings")` completely and added the new test case for an update which now correctly forces a new resource.